examples/node/pdf2svg.js

```diff
@@ -39,15 +41,12 @@ function writeToFile(svgdump, pageNum) {
 function getFileNameFromPath(path) {
   var index = path.lastIndexOf('/');
   var extIndex = path.lastIndexOf('.');
-  return path.substring(index , extIndex);
+  return path.substring(index, extIndex);
 }
 
 // Will be using promises to load document, pages and misc data instead of
 // callback.
-pdfjsLib.getDocument({
-  data: data,
-  disableNativeImageDecoder: true,
```

Comment:
Looks like disableNativeImageDecoder shall be replaced by different mode, e.g. nativeImageDecoderSupport = (none|store|decode)